ur_robot_driver/include/ur_robot_driver/rtde/rtde_client.h

```diff
@@ -48,6 +48,29 @@ namespace ur_driver
 {
 namespace rtde_interface
 {
+enum UrRtdeRobotStatusBits
```

Comment:
Please make this scoped enums as we use this all over the driver.

Reply:
Ok, then I'd rather switch to constants like

`static const uint8_t ROBOT_STATUS_BIT_IS_POWER_ON = 0;`

and so on, because accessing the bitsets with scoped enums gets quite ugly..

Reply:
I believe @fmauch already added a `std::underlying_type<>` somewhere?